example/src/main/java/io/netty/example/stomp/websocket/StompWebSocketProtocolCodec.java

```diff
@@ -52,13 +53,13 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
     }
 
     @Override
-    protected void encode(ChannelHandlerContext ctx, StompSubframe stompFrame, List<Object> out) throws Exception {
```

Comment:
This one still needs to propagate the exception from the frame encoder.

Reply:
fixed, missed this one  :(